services/brig/src/Brig/User/Search/Index.hs

```diff
@@ -257,6 +258,7 @@ createIndex ::
   [ES.UpdatableIndexSetting] ->
   -- | Number of shards
   Int ->
+  Maybe ES.TemplateName ->
```

Comment:
Can you please add a comment explaining that this template name is for a template to be deleted, not to be created as the name of the function `createIndex` might imply something like that.

Reply:
Done